lib/logstash/filters/syslog_pri.rb

```diff
@@ -105,11 +105,21 @@ def parse_pri(event)
 
     # Add human-readable names after parsing severity and facility from PRI
     if @use_labels
```

Comment:
```ruby
    return unless @use_labels
    
    if facility_label = @facility_labels[facility_code]
      event.set(@facility_label_key, facility_label)
    else
      event.tag('_syslogparsefailure')
      logger.debug? && logger.debug("Invalid facility code for event", :facility => facility_code)
      return
    end
    
    # severity code is inrange [0..7] by definition, no need to check any bound
    severity_label = @severity_labels[severity_code]
    event.set(@severity_label_key, severity_label) if severity_label
```

Reply:
I'll integrate this in #11 and close this as superseeded by it.